code-challenges.test.js

```diff
@@ -34,9 +56,32 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+describe("divisibleByThree", () => {
+    it("returns wether the number inside the object is evenly divisible by three", () => { 
+        expect(divisibleByThree(15)).toEqual("15 is divisible by three")
+        expect(divisibleByThree(0)).toEqual("0 is divisible by three")
+        expect(divisibleByThree(-7)).toEqual("-7 is not divisible by three")
```

Comment:
Careful to pay attention to what the prompt is asking, which is that the function should take an object as an argument. Here, you are giving the function a number as an argument.